sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,13 +7,15 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>3.2.1-preview</Version>
+		<Version>3.3.1-preview</Version>
```

Comment:
Update AzSdk.RP.props
AzureApiTag: This will be displayed as Nuget package tag

Reply:
Looks like it was already updated with this PR https://github.com/Azure/azure-sdk-for-net/pull/30491 - should I add anything else there?

Reply:
ok, then don't need to update